source/layouts/layout-pane.erb

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="robots" content="noindex, nofollow">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>GOV.UK Design System</title>
+    <%= stylesheet_link_tag :main, media: 'all' %>
+    <!--[if lt IE 9]><%= javascript_include_tag "ie.js" %><![endif]-->
+    <%= javascript_include_tag "application.js" %>
+  </head>
+  <body class="govuk-u-font-smoothing">
+    <a href="#content" class="govuk-c-skip-link">Skip to main content</a>
+    <div class="app-o-pane">
+      <div class="app-o-pane__header">
+        <%= partial 'components/header' %>
+      </div>
+      <div class="app-o-pane__body">
+        <div class="app-o-pane__toc">
```

Comment:
I wonder if this should be called `app-o-pane__aside` as it could contain other things than the table of content?